doc/src/sphinx/Futures.rst

```diff
@@ -168,7 +168,7 @@ The following retries a request infinitely should it fail with a
 
 	def fetchUrl(url: String): Future[HttpResponse]
```

Comment:
sorry, I should have been more clear.  could we change the return type to http.Response?  HttpResponse is a netty3 type, and finagle no longer exposes netty3 types in the public API, since we're trying to switch to netty4.  So the change here will look like:

```scala
def fetchUrl(url: String): Future[http.Response]
def fetchUrlWithRetry(url: String): Future[http.Response] =...
```

Does that make more sense?